plannerbenchmark/tests/test_integration.py

```diff
@@ -0,0 +1,122 @@
+import os
+from subprocess import Popen, DEVNULL
+import yaml
+import numpy as np
+import pytest
+
+
+def verify_resultDirExists():
+    resDirName = "tempResults"
+    resultFolderFound = False
+    for fname in os.listdir(resDirName):
+        if os.path.isdir(os.path.join(resDirName, fname)):
+            resultFolderFound = True
+            resultFolder = os.path.join(resDirName, fname)
+    # Check if folder time stamped folder exists
+    assert resultFolderFound
+    filenames = os.listdir(resultFolder)
+    assert "res.csv" in filenames
+    assert "planner.yaml" in filenames
+    assert "exp.yaml" in filenames
+    assert "plots" in filenames
+
+
+def verify_configFiles(caseFolder):
```

Comment:
As I understand, the config files are not being generated or modified as part of the runner execution or postprocessing, then why are they being compared?; the input configuration will always remain the same. Please correct me if I'm wrong in my understanding.

if that is the case, exp.yml and planner.yml should also not be in the results folder unlike the current scenario.

Reply:
Exactly, they should not be changed during the execution. However, it is often very tentative to change the content of the dictionary during running an experiment (to avoid the overhead of adding a new line to the yaml file). Then the config files would change during runtime and reproducibility is no longer guaranteed.
The last point is also the reason why the file is added to the result folder.

Reply:
Thanks for the clarification. Perhaps this point can be captured in the docstring for this test for future reference.

Reply:
Will address this in #15